### DISCUSSION TITLE
Influxdb@1 cli "missing"

### DISCUSSION THREAD

**Discussion by LukeSavage1306** [Homebrew/discussions#2660] [Casks]
Morning.  I have installed Influxdb@1 succesfully on a Mac running big sur and got the database service running OK.  But I can't seem to be able to start the cli (despite trying various permutations of instructions) which is essential to configuring the database.  Can anyone tell me if the cli is meant to install with the db, and if so how I am meant to be able to start it?
Thanks.

**Comment by gromgit** [comment]
~~Are you sure there's a CLI for Influx v1? The CLI for v2 is in Homebrew core (`influxdb-cli`), but [a quick search of InfluxData's GitHub repos](https://github.com/orgs/influxdata/repositories?language=&q=cli&sort=&type=all) nets only the [Go API for v1](https://github.com/influxdata/influxdb1-client).~~

Never mind, major thinko. `influxdb@1` is keg-only, so its `influx` CLI (which _does_ exist, now that I've tried installing it) isn't linked into the master Homebrew bin directory. You'll have to run it via `$(brew --prefix)/opt/influxdb@1/bin/influx`.

**Reply by LukeSavage1306** [reply]
Thanks grommet.  Still struggling though.  I have tried typing it directly as you suggested, just replacing the path that you have given to the path where influx seems to be on my machine so 

Luke$ brew install /usr/local/Cellar/influxdb@1/1.8.9/bin/influx

It does the usual Homebrew update checks and then comes back with:

Error: influx: /usr/local/Cellar/influxdb@1/1.8.9/bin/influx:1: invalid multibyte char (UTF-8)
So thinking it may already be installed as part of the influxdb@1 install I have tried

Luke$ brew services start /usr/local/Cellar/influxdb@1/1.8.9/bin/influx

to which I get the response

Error: influx: /usr/local/Cellar/influxdb@1/1.8.9/bin/influx:1: invalid multibyte char (UTF-8)
At a loss as to what to try next!

**Reply by SMillerDev** [reply]
Just run this in your terminal:

```

$(brew --prefix)/opt/influxdb@1/bin/influx --help
```

**Reply by LukeSavage1306** [reply]
So I did that - and it gave me lots of help messages -none of which seemed very helpful - but somewhere along the line something has changed and I can now run influx.  Not sure why - but hey its working.  So thanks to both gromgit and Smiller..  I just hope I don’t have to rebuild the machine for any reason as I will be back to square one! 🤪
